<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1894</width>
    <height>981</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="CitiesStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>881</width>
      <height>341</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>879</width>
       <height>339</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cities status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Actions in cities</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add city</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="city_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_city">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add city</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add stock</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="toward_city">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_stock">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add stock</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Product type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="product_type">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stock_capacity">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>510</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Product type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_product">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>570</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="product_type_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="product_name">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="toward_city_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add product</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Product name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>580</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cost</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cost">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="weight">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>530</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="toward_city_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>410</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>570</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add transport</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Product type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>640</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Move cost</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="transport_capacity">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>560</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="move_cost">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>630</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>460</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transport</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="transport_name">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_transport_btn">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>630</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add transport</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSolid">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>490</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Solid</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLiquid">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>510</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Liquid</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxGaseous">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>530</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaseous</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPassengersAvl">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>600</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Carry passangers</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="CitiesActions">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>40</y>
      <width>941</width>
      <height>341</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>939</width>
       <height>339</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>420</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="toward_city_4">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>410</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_transport_btn">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove transport</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>460</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transport</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPassengersAvl_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>530</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Carry passangers</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="transport_capacity_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>490</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="transport_name_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>500</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>390</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Remove transport</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>420</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City from</string>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="toward_city_5">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>410</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="replace_transport_btn">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>570</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace transport</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>500</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transport</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPassengersAvl_3">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>570</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Carry passangers</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="transport_capacity_3">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>530</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="transport_name_3">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>490</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>540</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>390</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Replace transport</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="toward_city_6">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>460</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City to</string>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="add_road">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>790</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add road</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="road_cost">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>790</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>800</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cost</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="city1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>640</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add road</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City1</string>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="city2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>680</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City2</string>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLand">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>720</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Land</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxWater">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>740</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Water</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAir">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>760</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Air</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Roadt type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>660</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City1</string>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="city1_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>650</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>700</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City2</string>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>630</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Transporting of production</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>740</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Product name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="city2_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>690</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="transport_production">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>810</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transporting</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="transporting_product_name">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>730</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="weight_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>770</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>780</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="count_passengers">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>810</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>820</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Get passengers</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1894</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
